web/app/js/components/util/Utils.js

```diff
@@ -119,10 +121,11 @@ export const toClassName = name => {
  Create regex string from user input for a filter
 */
 export const regexFilterString = input => {
+  let input_ = input;
   // make input lower case and strip out unwanted characters
-  input = input.replace(/[^A-Z0-9/.\-_*]/gi, '').toLowerCase();
+  input_ = input_.replace(/[^A-Z0-9/.\-_*]/gi, '').toLowerCase();
```

Comment:
I think you can transform these into a single liner:
```javascript
let input_ = input.replace(/[^A-Z0-9/.\-_*]/gi, '').toLowerCase();
```

Reply:
Done :)